### PR TITLE
codecept driver infra, test mods

### PR DESCRIPTION
**Pull request by lunaticusgreen**

### CODE REVIEW

**Comment by lunaticusgreen** [comment]
@puneet0191 PTAL @yurkovychv and I worked on some speed improvements and migrated to playwright accidentally, it gave us ~4 times faster e2e run, will it be ok if we also start using it in our "long" pipeline?

File: pmm-app/pr.codecept.json [hunk 1]
```diff
@@ -1,45 +1,19 @@
...
     },
-    "ResembleHelper" : {
-      "require": "codeceptjs-resemblehelper",
-      "screenshotFolder": "./tests/output/",
-      "baseFolder": "./tests/screenshots/base/",
-      "diffFolder": "./tests/screenshots/diff/"
-    },
-    "FileSystem": {}
+    "Grafana": {
+      "require": "./tests/helper/grafana_helper.js"
```
Comment (puneet0191) [inline pmm-app/pr.codecept.json:9]: Please add missing helper to PR

File: pmm-app/pr.codecept.json [hunk 2]
```diff
@@ -1,45 +1,19 @@
...
-          "enableLog": true,
-          "screenResolution": "2560x1600x24"
-        }
-      }
-    },
-    "SelenoidHelper": {
-      "require": "./tests/helper/selenoid_helper.js"
+    "Playwright": {
+      "url": "http://localhost/",
+      "browser": "chromium,"
```
Comment (puneet0191) [inline pmm-app/pr.codecept.json:6]: Change config to something like this: 
```
"Playwright": {
      "url": "http://localhost/",
      "show": true,
      "restart": true,
      "browser": "chromium",
      "waitForNavigation": "networkidle0",
      "windowSize": "1680x1240"
    },
```

File: pmm-app/tests/verifyInstanceIdAMISetup_test.js [hunk 3]
```diff
@@ -2,10 +2,10 @@ Feature('to run AMI Setup Page and verify Instance ID');
 
 Scenario(
   'Open the setup Page for AMI Instance @pmm-ami @visual-test',
-  async (I, adminPage, amiInstanceSetupPage, loginPage) => {
+  async (I, adminPage, amiInstanceSetupPage) => {
```
Comment (atymchuk) [inline pmm-app/tests/verifyInstanceIdAMISetup_test.js:5]: `adminPage` is not used here, consider removing from params

File: pmm-app/package.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
     "coverage": "grafana-toolkit plugin:test --coverage",
     "dev": "grafana-toolkit plugin:dev",
     "watch": "grafana-toolkit plugin:dev --watch",
-    "e2e": "codeceptjs run-multiple parallel --all --steps --grep '(?=.*)^(?!.*@visual-test)^(?!.*@not-pr-pipeline)' -c pr.codecept.json",
+    "e2e": "codeceptjs run-multiple parallel --all --steps --verbose --no-sandbox --grep '(?=.*)^(?!.*@visual-test)^(?!.*@not-pr-pipeline)' -c pr.codecept.json",
```
Comment (puneet0191) [inline pmm-app/package.json:12]: this is not the right way to set browser options.

File: pmm-app/tests/helper/grafana_helper.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    // remove if not used
+  }
+
+  // add custom methods here
+  // If you need to access other helpers
+  // use: this.helpers['helperName']
+
+  async Authorize() {
+    const { browserContext } = this.helpers.Playwright;
+    await browserContext.setExtraHTTPHeaders({ Authorization: 'Basic YWRtaW46YWRtaW4=' });
```
Comment (atymchuk) [inline pmm-app/tests/helper/grafana_helper.js:25]: Although this is for running the tests, but not really the best practice to hard-code the login/password. Can we pass via `.env` file?

**Pull request merged**